### PR TITLE
Improvement/triathlon startpage

### PR DESCRIPTION
**Pull request by BlueEnni**
@TomasKostadinov the stage is yours ;)

### CODE REVIEW

File: site/pages.json [hunk 1]
```diff
@@ -374,6 +374,15 @@
       "photoUrl": "/assets/images/running1.jpg",
       "description": "Triathlon",
       "children": [
+        {
+          "name": "Home",
```
Comment (TomasKostadinov) [inline site/pages.json]: Can we change this to Triathlon Home?

**Review by TomasKostadinov — CHANGES_REQUESTED** [review]
Looks good otherwise

File: site/pages.json (hunk 1 above)
Comment (BlueEnni) [reply site/pages.json]: Done

**Review by BlueEnni — COMMENTED** [review]
can be merged now

**Review by BlueEnni — COMMENTED** [review]
.

File: site/design/index.html [hunk 2]
```diff
@@ -309,7 +309,7 @@ <h2>Abteilung</h2>
 
         <ul class="value-grid">
             <li class="item">
-                <div class="key">
+                <div>
```
Comment (TomasKostadinov) [inline site/design/index.html:312]: why?
Just use the same html als everywhere else. removing this class breaks it on mobile.

File: site/lauftreff/index.html [hunk 3]
```diff
@@ -124,80 +124,94 @@ <h4>3.2 Stabilisationstraining</h4>
...
-                            <td>Rumpf, Hüfte, Oberschenkel, Füße</td>
-                            <td>Sporthalle Reutenen</td>
-                        </tr>
-                        </tbody>
-                    </table>
+                <h2 id="trainingszeiten" class="scroll-table">Trainigszeiten</h2>
+                <div>
+                    <ul class="value-grid">
+                        <li class="item">
+                            <div>
```
Comment (TomasKostadinov) [inline site/lauftreff/index.html]: Same here

File: site/leichtathletik/index.html [hunk 4]
```diff
@@ -23,7 +23,7 @@ <h1>Trainingszeiten</h1>
         <h2>Schülerleichtathletik</h2>
         <ul class="value-grid">
             <li class="item">
-                <div class="key">
```
Comment (TomasKostadinov) [inline site/leichtathletik/index.html]: And here

File: site/pages.json [hunk 5]
```diff
@@ -401,18 +399,7 @@
           "title": "mobi-stabi-training",
```
Comment (TomasKostadinov) [inline site/pages.json]: This is the browser title - doesn't look good IMO.
Bildschirmfoto 2019-11-27 um 12 34 22

File: site/pages.json [hunk 6]
```diff
@@ -421,18 +408,7 @@
           "title": "radfahren",
```
Comment (TomasKostadinov) [inline site/pages.json]: Same here

File: site/pages.json [hunk 7]
```diff
@@ -441,18 +417,7 @@
       "title": "schwimmen",
```
Comment (TomasKostadinov) [inline site/pages.json]: And here

File: site/pages.json [hunk 8]
```diff
@@ -441,18 +417,7 @@
...
       "photoUrl": "/assets/images/running1.jpg",
       "description": "schwimmen",
-      "children": [
-        {
-        "name": "schwimmen",
-        "destination": "trainingsplan",
-        "file": "triathlon/schwimmen/trainingsplan/index.html",
-        "title": "schwimmen",
```
Comment (TomasKostadinov) [inline site/pages.json:612]: And here

File: site/triathlon/schwimmen/index.html [hunk 9]
```diff
@@ -1,143 +1,137 @@
...
+            schwimmen wir zusätzlich im Waldbad Heidenheim auf der 50 Meter
+            Bahn. Treffpunkt ist mittwochs um 18.30 Uhr im Becken. Den Eintritt
+            übernimmt die Abteilung. Im Winter haben wir dienstags um 17:45 Uhr
+            eine 25 m-Bahn im Jurawell in Herbrechtingen. Hier müsst ihr den
+            Eintritt selbst übernehmen. Es empfiehlt sich eine Geldwertkarte.
+            Trainer sind Anka Röhr, Sven Müller und Michael Bacher. Je nach
+            Verfügbarkeit der Trainer findet auch mal nur ein freies Training
+            statt.
+            <br><br>
+          </p>
```
Comment (TomasKostadinov) [inline site/triathlon/schwimmen/index.html:36]: No need to write custom css and HTML. 
I appreciate the effort but we wanted to have a unified design that looks the same through the whole website. The goal is to have a continious ui and user experience.
Also, the text break is weird on mobile
Bildschirmfoto 2019-11-27 um 12 40 07

**Review by TomasKostadinov — CHANGES_REQUESTED** [review]

File: site/pages.json (hunk 7 above)
Comment (BlueEnni) [reply site/pages.json]: With those change requests I can work with

File: site/triathlon/schwimmen/index.html (hunk 9 above)
Comment (BlueEnni) [reply site/triathlon/schwimmen/index.html:36]: I agree with the weird text break - I'm trying to solve that

Comment (BlueEnni) [reply site/triathlon/schwimmen/index.html:36]: fixed weird break

File: site/pages.json (hunk 8 above)
Comment (BlueEnni) [reply site/pages.json:612]: changed to "Triathlon"

File: site/pages.json (hunk 7 above)
Comment (BlueEnni) [reply site/pages.json]: changed to "Triathlon"

File: site/pages.json (hunk 6 above)
Comment (BlueEnni) [reply site/pages.json]: changed to "Triathlon"

File: site/pages.json (hunk 5 above)
Comment (BlueEnni) [reply site/pages.json]: changed to "Triathlon"

File: site/leichtathletik/index.html (hunk 4 above)
Comment (BlueEnni) [reply site/leichtathletik/index.html]: added the class key

File: site/lauftreff/index.html (hunk 3 above)
Comment (BlueEnni) [reply site/lauftreff/index.html]: added the class key

File: site/design/index.html (hunk 2 above)
Comment (BlueEnni) [reply site/design/index.html:312]: added the class key

**Review by TomasKostadinov — APPROVED** [review]

**Pull request merged**